session/README.md

```diff
@@ -8,7 +8,7 @@ gorails/session
 With Go and git installed:
 
 ```
-go get -u code.google.com/p/go.crypto/pbkdf2
```

Comment:
I believe these days this line is redundant, since the `go get` resolves and downloads package dependencies. Could you simply remove this line?